build.gradle

```diff
@@ -44,6 +44,8 @@ repositories {
 }
 
 dependencies {
+    implementation 'org.projectlombok:lombok:1.18.30'
+    annotationProcessor 'org.projectlombok:lombok:1.18.30'
     implementation 'org.springframework.boot:spring-boot-starter'
```

Comment:
We already have Lombok dependency added, please take a look at lines:
https://github.com/josdem/jmetadata/blob/9105b7256fad7e543ce1312d0d8ef8f50239485d/build.gradle#L57L58